sitecore-packages.json

```diff
@@ -234,5 +234,13 @@
   "Sitecore Publishing Module 9.3.0.0 rev. r00546.2197.scwdp.zip": {
     "url": "",
     "hash": ""
+  },
+  "Sitecore Publishing Module 10.0.0.0 rev. r00565.2578.scwdp.zip": {
```

Comment:
Hi @bplasmeijer,

This package filename is not correct and is causing the build to fail. The correct package filename is "Sitecore Publishing Module 10.0.0.0 rev. r00568.2697.scwdp.zip".

Reply:
Also it seems that the correct package is available here: https://dev.sitecore.net/Downloads/Sitecore_Publishing_Service_Module/10x/Sitecore_Publishing_Service_Module_1000.aspx

This is the complete package definition:

    "Sitecore Publishing Module 10.0.0.0 rev. r00568.2697.scwdp.zip": {
        "url": "https://dev.sitecore.net/~/media/A06BC5BBBCA84F2F90AC08CB456A3801.ashx",
        "hash": "E1088A6553B82AA0346264D7756B021F4FFBE7293F475E09A3E9415E3D16C860"
    }

Reply:
Thanks for the feedback @afaniuolo 
We are going to add the ```experimental``` flag to the PS images.
You need to use the [Sitecore Azure Toolkit](https://dev.sitecore.net/Downloads/Sitecore_Azure_Toolkit/2x/Sitecore_Azure_Toolkit_250.aspx), and generate ```Publishing Module 10.0.0.0 rev. r00568.2697.scwdp.zip```

Reply:
The module from dev.sitecore.com is a zip and not an scwdp.zip. Unfortunately it needs to be converted (manually) to an scwdp which can be done using the Sitecore Azure Toolkit.

Ensure you install the [Sitecore Azure Toolkit](https://dev.sitecore.net/Downloads/Sitecore_Azure_Toolkit/2x/Sitecore_Azure_Toolkit_250.aspx) correctly (with prerequisites) and then:

```powershell
Import-Module <path-to-sat>\tools\Sitecore.Cloud.Cmdlets.dll
ConvertTo-SCModuleWebDeployPackage -Path <path to Sitecore zip>
```

Reply:
Ah, thanks! I thought that I was missing something there. :-)

The `ConvertTo-SCModuleWebDeployPackage` command has been deprecated and removed in SAT 2.5, but it worked using SAT 2.4.